gammapy/estimators/ts_map.py

```diff
@@ -773,6 +847,8 @@ def _ts_value(position, counts, exposure, background, kernel, norm, flux_estimat
     norm : `~numpy.ndarray`
         Norm image. The flux value at the given pixel position is used as
         starting value for the minimization.
+    name : str
+        Name of the associated Dataset (optional)
```

Comment:
Why do you add this? Is it needed at all?

Reply:
In order to trace where an error occurs.